activesupport/lib/active_support/benchmark.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  # = \Benchmark
+  module Benchmark
```

Comment:
This should be `# :nodoc:` I don't want people using it.

Reply:
Yeah, sure. Updated deprecation message/changelog